hack/lib/serverless.bash

```diff
@@ -145,29 +145,34 @@ function deploy_knativeserving_cr {
   # Wait for the CRD to appear
   timeout 900 "[[ \$(oc get crd | grep -c knativeservings) -eq 0 ]]"
 
-  local rootdir
-  rootdir="$(dirname "$(dirname "$(dirname "$(realpath "${BASH_SOURCE[0]}")")")")"
-
   # Install Knative Serving
   # Deploy the full version of KnativeServing (vs. minimal KnativeServing). The future releases should
   # ensure compatibility with this resource and its spec in the current format.
   # This is a way to test backwards compatibility of the product with the older full-blown configuration.
-  oc apply -n "${SERVING_NAMESPACE}" -f "${rootdir}/test/v1alpha1/resources/operator.knative.dev_v1alpha1_knativeserving_cr.yaml"
-
   if [[ $FULL_MESH == "true" ]]; then
-    enable_net_istio
+    deploy_with_istio
+  else
+    deploy_with_kourier
   fi
 
   oc wait --for=condition=Ready knativeserving.operator.knative.dev knative-serving -n "${SERVING_NAMESPACE}" --timeout=900s
 
   logger.success 'Knative Serving has been installed successfully.'
 }
 
-# enable_net_istio adds patch to KnativeServing:
+function deploy_with_kourier {
+  local rootdir
+  rootdir="$(dirname "$(dirname "$(dirname "$(realpath "${BASH_SOURCE[0]}")")")")"
+  oc apply -n "${SERVING_NAMESPACE}" -f "${rootdir}/test/v1alpha1/resources/operator.knative.dev_v1alpha1_knativeserving_cr.yaml"
```

Comment:
[tests](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift-knative_serverless-operator/1378/pull-ci-openshift-knative-serverless-operator-main-4.9-operator-e2e-aws-ocp-49/1483365006010486784/build-log.txt) fail here as manifest is empty.